Add troubleshooting section for onewire temperature sensors

epenet opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Linked to https://github.com/home-assistant/core/issues/150388


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

epenet committed:
Add troubleshooting section for temperature sensors

epenet commented:
cc @Fkuebler / @migube / @szczukot

c0ffeeca7 commented on source/_integrations/onewire.markdown:
```diff
@@ -148,3 +148,13 @@ Remember to replace `<sensor_name>` with the actual name of the sensor, as seen
+
+### 85°C temperature
+
+85°C is the power-on reset value of the `DS18B20`. It can report such a value when its tiny internal capacitor that acts as its power supply is depleted.
```
I think copilot has a point, in both cases. If you want to keep the original explanation for the more technical audiance, could you add the simpler explanation also?

epenet committed:
Update troubleshooting section for DS18B20 temperature

Clarify the meaning of 85°C readings from the DS18B20 sensor and provide troubleshooting steps.

epenet committed:
Update onewire.markdown

epenet committed:
Update onewire.markdown

c0ffeeca7 reviewed (approved):
Thank you, @epenet :+1:
